### PR TITLE
[NOT TO MERGE] post about transitioning from jbuilder to dune

### PR DESCRIPTION
**Pull request by diml**
I'm opening this PR here just to review this post, then I'll post it on discuss.

### CODE REVIEW

File: post.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Dune 1.0.0 is coming soon, what about jbuilder projects?
+
+The 1.0.0 release of dune is planned for next week. It will be the
+first release of jbuilder as dune, so what to expect for existing
+jbuilder projects?
+
+The good news is that dune is fully backward compatible with
+jbuilder. The `dune` package in opam will install both `dune` and
+`jbuilder` binaires and both will be usable on existing jbuilder
```
Comment (emillon) [inline post.md]: binaires -> binaries

File: post.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+projects.
+
+We do want to stop supporting jbuilder projects eventually, so the
+plan is to continue to support the `jbuilder` binary and `jbuild`
+files for another year. 6 months from now dune will start displaying
+warnings when using the `jbuilder` binary or `jbuild` files. A more
+detailied [migration plan][mp] is available in the manual. In
+particular in contains a precise list of things to do to migrate a
```
Comment (emillon) [inline post.md]: in -> it

File: post.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+This section is not an exhaustive list of changes, but it describes
+the important changes to expect when switching from `jbuilder` to
+`dune`.
+
+### Versioning and configuration files
+
+A big part of the work in Dune 1.0.0 was getting the versioning story
+right. With jbuilder it was hard for us to make the language evolve
+without breaking the build of existing projects. Dune can handle that
+gracefuly and in the future it will be much easier to introduce
```
Comment (emillon) [inline post.md]: gracefuly -> gracefully

File: post.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+breaking changes.
+
+The versioning of all dune file in a project is controlled via a
```
Comment (emillon) [inline post.md]: file -> files

File: post.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+While this file is required in order to ensure compatibility with
+future versions of Dune, it is not required to write one by hand every
+time you start a new project or want to do a quick experiment. Dune
+will create and edit this file for you as needed, so simply start by
+creating a `dune` file (the new `jbuild` files) and dune will take
+care of the rest.
+
+Note that the syntax inside `dune`, `dune-project`, ... files is
+slightly different from the one insde `jbuild` files. In particular
```
Comment (emillon) [inline post.md]: insde -> inside

File: post.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+the language requires many less parentheses and the syntax of
```
Comment (emillon) [inline post.md]: I think it's "fewer" here.

File: post.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+While this file is required in order to ensure compatibility with
+future versions of Dune, it is not required to write one by hand every
+time you start a new project or want to do a quick experiment. Dune
+will create and edit this file for you as needed, so simply start by
+creating a `dune` file (the new `jbuild` files) and dune will take
+care of the rest.
+
+Note that the syntax inside `dune`, `dune-project`, ... files is
+slightly different from the one inside `jbuild` files. In particular
+the language requires fewer parentheses and the syntax of varialbes
```
Comment (xclerc) [inline post.md]: Typo: "varialbes" -> "variables".

File: post.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+The good news is that dune is fully backward compatible with
+jbuilder. The `dune` package in opam will install both `dune` and
+`jbuilder` binaries and both will be usable on existing jbuilder
+projects.
+
+We do want to stop supporting jbuilder projects eventually, so the
+plan is to continue to support the `jbuilder` binary and `jbuild`
+files for another year. 6 months from now dune will start displaying
+warnings when using the `jbuilder` binary or `jbuild` files. A more
+detailied [migration plan][mp] is available in the manual. In
```
Comment (rgrinberg) [inline post.md]: Is this going to be a proper link?

**Comment by diml** [comment]
Thanks for the reviews, I'm posting this to discuss now

**Pull request closed without merging**